admin_api.go

```diff
@@ -181,6 +183,64 @@ func (a *AdminAPI) AccessKeys(ctx context.Context, accessKeyOpts ...queries.Admi
 	return res, nil
 }
 
+// Paymails retrieves a paginated list of paymail addresses via the Admin Paymails API.
+// The response includes user paymails along with pagination metadata, such as
+// the current page number, sort order, and the field used for sorting (sortBy).
+//
+// Query parameters can be configured using optional query options. These options allow
+// filtering based on metadata, pagination settings, or specific paymail attributes.
+//
+// The API response is unmarshaled into a *queries.PaymailAddressPage struct.
+// Returns an error if the API request fails or the response cannot be decoded.
+func (a *AdminAPI) Paymails(ctx context.Context, opts ...queries.PaymailQueryOption) (*queries.PaymailAddressPage, error) {
+	res, err := a.paymailsAPI.Paymails(ctx, opts...)
+	if err != nil {
+		return nil, paymails.HTTPErrorFormatter("failed to retrieve paymail addresses page", err).FormatGetErr()
+	}
+
+	return res, nil
+}
+
+// Paymail retrieves the paymail address associated with the specified ID via the Admin Paymails API.
+// The response is expected to be unmarshaled into a *response.PaymailAddress struct.
+// Returns an error if the request fails or the response cannot be decoded.
+func (a *AdminAPI) Paymail(ctx context.Context, ID string) (*response.PaymailAddress, error) {
+	res, err := a.paymailsAPI.Paymail(ctx, ID)
+	if err != nil {
+		msg := fmt.Sprintf("failed retrieve paymail address with ID: %s", ID)
+		return nil, paymails.HTTPErrorFormatter(msg, err).FormatGetErr()
+	}
+
+	return res, nil
+}
+
+// CreatePaymail creates a new paymial address record via the Admin Paymails API.
+// The provided command contains the necessary parameters to define the paymail address record.
+//
+// The API response is unmarshaled into a *response.Xpub PaymailAddress.
```

Comment:
```suggestion
// The API response is unmarshalled into a *response.Xpub PaymailAddress.
```
Please replace other words with this typo.